File: Backend/Sales/Sales/Models/Invoice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Sales.Models
{
    public class Invoice
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public int Invoice_ID { set; get; }
        public DateTime Invoice_Date { set; get; }
        [ForeignKey("Customer")]
        public int CX_ID { set; get; }
        public double Total_Price { set; get; }
        public int Total_Quantity { set; get; }
        public Customer Customer { set; get; }
        public List<Item> Items { set; get; }

    }
}

File: Frontend/Sales/src/app/_models/invoice.ts
export class Invoice {
  constructor(
    public  invoice_ID ?:number ,
    public  invoice_Date?:Date,
    public  cX_ID?:number  ,
    public  cX_Name?:string ,
    public  total_Price?:number  ,
    public  total_Quantity?:number ,

  ){}
}
